### PR TITLE
Updated the README with new Carthage instructions. Also updated a bunch of old content.

### PR DESCRIPTION
**Pull request by alda-optimizely**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,50 +2,63 @@
...
+This repository houses the Objective-C SDK for Optimizely X Mobile.
 
 ## Getting Started
 
 ### Using the SDK
 
-See the Optimizely server-side testing [developer documentation](https://developers.optimizely.com/x/solutions/sdks/reference/index.html?language=objectivec) to learn how to set
-up your first custom project and use the SDK. **Please note that you must be a member of the private server-side testing beta to create custom projects and use this SDK.**
+See the Optimizely X Mobile [developer documentation](https://developers.optimizely.com/x/solutions/sdks/reference/index.html?language=objectivec) to learn how to set
+up your first custom project and start using the SDK.
```
Comment (wangjoshuah) [inline README.md]: we can remove "custom"

File: README.md [hunk 2]
```diff
@@ -2,50 +2,63 @@
...
 
 ### Requirements
 * iOS 8.0+ / tvOS 9.0+
-* Foundation.framework
```
Comment (wangjoshuah) [inline README.md]: do we no longer use Foundation.framework?

**Review by wangjoshuah — APPROVED** [review]
looks good just one small nit

File: README.md (hunk 2 above)
Comment (alda-optimizely) [reply README.md]: yea all apps need it though, so no need to list it as a requirement

**Comment by wangjoshuah** [comment]
Can you update the reference section of the dev docs after this is published? 
https://github.com/optimizely/developers.optimizely.com/commit/8ee73e9703b45ef5919d4ccecfeaa1b187f571c8

**Pull request merged**